config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -29,6 +29,7 @@ presubmits:
         - --build=bazel
         - --cluster=
         - --extract=local
+        - --flush-mem-after-build=true
```

Comment:
Can you also open another PR to adjust what was done in: https://github.com/kubernetes/test-infra/pull/18698 (to make everything more consistent).

Reply:
As I understand you are referring to files like `config/jobs/kubernetes/sig-release/release-branch-jobs/1.19.yaml`?

Sure, after we will have a consensus here, I will 'backport' changes to release-branch-jobs. SG?

Reply:
and other release branches yes